lib/presentation/extensions/gps_coordinates_extensions.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:latlong2/latlong.dart';
+import 'package:movna/domain/entities/gps_coordinates.dart';
+
+extension GpsCoordinatesExt on GpsCoordinates {
+  LatLng get latLng => LatLng(latitudeInDegrees, longitudeInDegrees);
```

Comment:
I prefer being explicit on the fact that this is a conversion :
```suggestion
  LatLng toLatLng() => LatLng(latitudeInDegrees, longitudeInDegrees);
```